### ISSUE TITLE
Seems Lunr.js doesn't support other language

### ISSUE THREAD

**Issue by mashirozx** [slashdotdash/jekyll-lunr-js-search#119]
When I search for English words, it works, but when I use Chinese or Japanese, it doesn't work. However, my blog is mainly in these two languages. 

I guess it is Lunr.js doesn't support. 

Was there any alternative？😂

**Comment by olivernn** [comment]
Lunr.js definitely does support [other languages](https://github.com/MihaiValentin/lunr-languages). I don't know how that would be integrated into this project though.

**Comment by mashirozx** [comment]
@olivernn Yep, seems a lot of works to make this search engine support other languages
